gensim/__init__.py

```diff
@@ -4,11 +4,12 @@
 
 """
 
+__version__ = '4.0.0beta'
```

Comment:
Tangential comment: As a matter of labeling-correctness, '4.0.0beta' is already released and frozen, so adding this label now can't possibly be a correct description of code-from-this-branch. What I've seen as a useful process before - unless some sort of auto process is auto-labeling the code tree on every test build - is to immediately promote this identifier to some ".dev"/etc extension of the last version, or preview of the next-expected-version, so that a precise release label like this only ever exists in the repo for the brief moment that it's true, and the code is feeding a build-for-release.

Reply:
Yes, .dev1, .dev2, etc builds are helpful.

Reply:
You're right. Should have been incremented right after the beta release according to [our own instructions](https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#update-gensim-version-to-report-the-next-release).

I'll update it in this PR.